monkey/infection_monkey/pe/actions/__init__.py

```diff
@@ -0,0 +1,27 @@
+from abc import ABCMeta, abstractmethod
+from os.path import dirname, basename, isfile, join, realpath
+import glob
+
+__author__ = 'D3fa1t'
+
+
+class HostPrivExploiter(object):
```

Comment:
I feel like `PrivEscalator` would be a better name (mainly the Host part is confusing), but maybe it's just me.